lib/elixir/pages/Behaviours.md

```diff
@@ -0,0 +1,52 @@
+# Behaviours
+
+Behaviours in Elixir (and Erlang) are a way to separate and abstract the generic part of a component (which becomes the *behaviour module*) from the specific part (which becomes the *callback module*).
```

Comment:
Is this perhaps too specific? It is indeed generally how behaviours are used, but perhaps we should describe them in terms of being a specification for an interface?

I don't think much would be lost leaving it as is, but just wanted to bring this up. :)